src/python/pants/backend/python/dependency_inference/subsystem.py

```diff
@@ -166,3 +166,27 @@ class PythonInferSubsystem(Subsystem):
             """
         ),
     )
+
+    ignore_unowned_imports = StrListOption(
```

Comment:
This is a bikeshed, feel free to disregard:
this name reads to me like a boolean flag, doing what `unowned_dependency_behavior` would do. I can't really find a good alternative, though. Maybe `ignored_unowned_imports`, `ignorable_unowned_imports`, or maybe just `unowned_imports`? or maybe more like the pragma with `non_inferable_dependencies`?

Reply:
No, this is actually very helpful, thank you. I like `ignored_unowned_imports` a lot more, this is what we now have :)

Reply:
No, that's actually rather helpful! I like `ignored_unowned_imports` a lot and have changed the option name. Thank you!